Sources/main.cpp

```diff
@@ -120,7 +228,13 @@ void PrintUsage(const char *programName)
             "  A directory, indicating that all files and subdirectories \n"
             "    of that directory are included in the package, or\n"
             "  A file name, indicating that the file is included in the \n"
-            "    root of the package.\n"
+            "    root of the package, or\n"
```

Comment:
With MakeAppx, you can *either* specify a directory *or* specify a mapping file. fb-util-for-appx allows for both to be mixed.

https://msdn.microsoft.com/en-us/library/windows/desktop/hh446767(v=vs.85).aspx